test/CMakeLists.txt

```diff
@@ -230,10 +239,19 @@ foreach(filename IN LISTS XTENSOR_TESTS)
 endforeach()
 
 add_executable(test_xtensor_lib ${COMMON_BASE} ${XTENSOR_TESTS} ${XTENSOR_HEADERS})
+if(XTENSOR_USE_XSIMD)
+    target_compile_definitions(test_xtensor_lib PRIVATE XTENSOR_USE_XSIMD)
+    target_link_libraries(test_xtensor_lib PRIVATE xsimd)
+endif()
+if(XTENSOR_USE_TBB)
+    target_compile_definitions(test_xtensor_lib PRIVATE XTENSOR_USE_TBB)
+    target_include_directories(test_xtensor_lib PRIVATE ${TBB_INCLUDE_DIRS})
+    target_link_libraries(test_xtensor_lib PRIVATE ${TBB_LIBRARIES})
+endif()
 target_include_directories(test_xtensor_lib PRIVATE ${XTENSOR_INCLUDE_DIR})
+target_link_libraries(test_xtensor_lib xtensor ${GTEST_BOTH_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
This command should remain after adding additional dependencies

Reply:
I placed it consistently with line 232 above in the loop for all tests. Should I change both blocks consistently to move the line down?

Reply:
I changed `test/CMakeLists.txt` to consistently move the `target_include_directories` and `target_link_libraries` below the optional block `if(DOWNLOAD_GTEST OR GTEST_SRC_DIR)`. I hope this is clean and consistent?

Reply:
Yes!